evm-types.md

```diff
@@ -682,10 +686,10 @@ Addresses
 -   `#lookup` looks up a key in a map and returns 0 if the key doesn't exist, otherwise returning its value.
 
 ```k
-    syntax Int ::= #lookup ( Map , Int ) [function]
- // -----------------------------------------------
-    rule [#lookup.some]: #lookup( (KEY |-> VAL) M, KEY ) => VAL
-    rule [#lookup.none]: #lookup(               M, KEY ) => 0 requires notBool KEY in_keys(M)
+    syntax Int ::= #lookup ( Map , Int ) [function, functional]
+ // -----------------------------------------------------------
+    rule [#lookup.some]: #lookup( (KEY |-> {VAL}:>Int) M, KEY ) => VAL
+    rule [#lookup.none]: #lookup(                      M, KEY ) => 0 requires notBool KEY in_keys(M)
```

Comment:
We should make the second rule `owise`, to cover the case where `VAL` is not an `Int`, if we want to make this total.

Reply:
replaced with `#Ceil`